kleinanzeigen_bot/resources/categories.yaml

```diff
@@ -18,6 +18,9 @@ Software: 161/225/software
 Telefone: 161/173/telefone
 Tastaturen: 161/225/tastatur_maus
 
+#Auto, Rad & Boot
+Reifen: 10/223
```

Comment:
I think the value should be `210/223/reifen_felgen`. Could you also change the name from `Reifen` to the more specific term `Autoreifen`?